LLama.Unittest/MemoryDisposalTests.cs

```diff
@@ -1,12 +1,39 @@
 ﻿using LLama.Common;
+using System.Runtime.InteropServices;
+using System.Text;
 
 namespace LLama.Unittest;
 
 public class MemoryDisposalTests
 {
+    [DllImport("llama", CallingConvention = CallingConvention.Cdecl)]
+    public static extern int ggml_backend_cuda_get_device_count();
+
+    [DllImport("llama", CallingConvention = CallingConvention.Cdecl)]
+    public static extern void ggml_backend_cuda_get_device_description(int device, StringBuilder description, ulong description_size);
+
+    [DllImport("llama", CallingConvention = CallingConvention.Cdecl)]
+    public static extern void ggml_backend_cuda_get_device_memory(int device, out ulong free, out ulong total);
+
+    private static ulong GetGPUFreeMemory()
+    {
+        ulong freemem = 0;
+        int dc = ggml_backend_cuda_get_device_count();
+        if (dc > 0)
+        {
+            ulong totalmem = 0;
+            StringBuilder sb = new StringBuilder(1024 * 1024);
```

Comment:
Is this `StringBuilder` bit necessary? Looks like it might be a leftover from debugging?

Reply:
To be clear, talking about these specific lines:
```csharp
StringBuilder sb = new StringBuilder(1024 * 1024);
ggml_backend_cuda_get_device_description(0, sb, 1024 * 1024);
```
which appear to get a string, and then do nothing with it!

Reply:
yes, you are right, I have removed it